.drone.yml

```diff
@@ -4242,7 +4256,8 @@ type: kubernetes
 name: build-docker-images
 
 environment:
-  RUNTIME: go1.17.2
+  BUILDBOX_VERSION: "teleport9"
```

Comment:
You mean `teleport10`, right?

Reply:
Bah. Apparently `dronegen` doesn't automatically update that one. *sigh*